tests/unit/services/user-activity-test.js

```diff
@@ -58,6 +58,32 @@ test('disableEvent', function (assert) {
   service.disableEvent(event);
 
   assert.ok(!service.get('enabledEvents').includes(event), 'removed event from enabledEvents');
+  assert.notOk(service._eventsListened.includes(event), 'event should not be listed as listened');
+});
+
+test('re-enabled events should fire', function (assert) {
+  let event = 'foo';
+  let service = this.subject({
+    enabledEvents: emberArray(),
+    _setupListeners: this.stub()
+  });
+
+  this.stub(window, 'addEventListener');
+
+  assert.notOk(service.get('enabledEvents.length'), 'enabledEvents preserved on init');
+
+  service.enableEvent(event);
+  assert.ok(window.addEventListener.called, 'event was not handled');
```

Comment:
`this.stub` actually returns the stub object, so that you don't have to go access `window.addEventListener` to check things like `called`